### PR TITLE
Some changes about the calculator parts in UG.

### PR DESCRIPTION
**Pull request by yuqiaoluolong**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -7,5 +7,5 @@ Display | Name | Github Profile | Portfolio
 ![](https://via.placeholder.com/100.png?text=Photo) | John Doe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
 ![](https://via.placeholder.com/100.png?text=Photo) | Don Joe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
 ![](https://via.placeholder.com/100.png?text=Photo) | Ron John | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | John Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Liu Haoran | [Github](https://github.com/yuqiaoluolong) | [Portfolio](docs/team/yuqiaoluolong.md)
```
Comment (HengFuYuen) [inline docs/AboutUs.md]: Perhaps you might need to pull in the new merge and update this file. When docs/team/filename.md is used the website cannot be published. Therefore consider changing it to team/filename.md.

**Review by HengFuYuen — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (yuqiaoluolong) [reply docs/AboutUs.md]: OK

Comment (yuqiaoluolong) [reply docs/AboutUs.md]: I deleted the changes and may create a new PR later for that.

**Review by HengFuYuen — APPROVED** [review]
LGTM

**Pull request merged**